framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2955,7 +2955,21 @@ private AnnotationMirror getDeclAnnotation(
 
         Set<AnnotationMirror> results = AnnotationUtils.createAnnotationSet();
         // Retrieving the annotations from the element.
-        results.addAll(elements.getAllAnnotationMirrors(elt));
+        List<? extends AnnotationMirror> fromEle = elements.getAllAnnotationMirrors(elt);
+        for (AnnotationMirror annotation : fromEle) {
+            try {
```

Comment:
Do you know why the CompletionFailure can happen here?
Should we put this try/catch into a helper method and use it everywhere?